features/gateway_injection.md

```diff
@@ -0,0 +1,146 @@
+# Feature: Gateway Injection
```

Comment:
My intention was for the 

# Feature to just be a heading, but everyone is adding the feature name after it. I think it might make sense to change the template and remove `**Feature name:**`